src/styles.css

```diff
@@ -0,0 +1,1174 @@
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.button-click {
+  position: static;
+  width: 70%;
+  height: 72px;
+  left: 0.5px;
+  top: 216px;
+  flex: none;
+  order: 3;
+  flex-grow: 0;
+}
+
+.button-resume {
+  position: static;
+  width: 70%;
+  height: 72px;
+  left: 0.5px;
+  top: 216px;
+  flex: none;
+  order: 3;
+  flex-grow: 0;
+  margin: 24px 0.5px;
+  padding-left: 20px;
+}
+
+.btn {
+  flex-direction: row;
+  align-items: center;
+  background: #fff;
+  padding: 7px;
+  top: 12px;
+  border: 1px solid #6070ff;
+  box-sizing: border-box;
+  border-radius: 8px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  margin-top: 5%;
+}
+
+.btn:hover {
+  background-color: #396df2;
+  color: #fff;
+}
+
+button:active {
+  background-color: #2230d2;
+}
+
+.btn:active {
+  background-color: #2230d2;
+}
+
+.about {
+  background: #fff;
+  width: 70%;
+  padding: 64px;
+  border-top-right-radius: 55px;
+}
+
+.about-me {
+  display: flex;
+  position: static;
+  width: 40%;
+  left: 0.5px;
+  top: 0.5px;
+  right: auto;
+  padding: 15px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40.5px;
+  line-height: 52px;
+  align-items: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 12px 0.5px;
+}
+
+.lang {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 90%;
+  padding: 24px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+}
+
+.imglang {
+  width: 30px;
+  padding-right: 40px;
+}
+
+.lang-title {
+  position: absolute;
+  height: 24px;
+  left: 12px;
+  right: 0;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20.5px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  color: #000;
+  padding-left: 24px;
+  justify-content: space-between;
+}
+
+.language-list {
+  padding: 5px;
+}
+
+.lang-skill {
+  position: static;
+  width: 80%;
+  height: 66.72px;
+  left: 12px;
+  top: 12px;
+  display: flex;
+  padding-left: 20px;
+  align-items: center;
+  column-gap: 20px;
+  background: #f7f7f9;
+  border-radius: 8px;
+  margin: 12px 0;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #253858;
+}
+
+.line {
+  display: flex;
+  height: 2px;
+  width: 90%;
+  background: #dfe1e6;
+}
+
+.contact-me {
+  background-color: #6070ff;
+  border: 1px solid #cfd8dc;
+  border-radius: 100px 0 0 0;
+  margin-top: 0%;
+  width: 98%;
+}
+
+.bg-image {
+  background-image: url(../assets/contact-me.png);
+  background-repeat: no-repeat;
+  background-position-x: right;
+  background-size: contain;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+}
```

Comment:
- Hey, let's use the correct background image of the form for the desktop version as seen in the Figma design:


*Figma design*
![Screenshot from 2023-05-02 00-08-02](https://user-images.githubusercontent.com/59397159/235546990-7d6dbed2-0e16-4dde-95c8-6fdd815491f0.png)


*your design*
![Screenshot from 2023-05-02 00-11-56](https://user-images.githubusercontent.com/59397159/235547168-25cb220d-13d3-4b7c-9a49-45a0a63c23c7.png)